docker/Dockerfile

```diff
@@ -41,9 +41,10 @@ RUN groupadd -r tigris && useradd -r -s /bin/false -g tigris tigris
 RUN mkdir -p /server /etc/tigrisdata/tigris /etc/foundationdb/
 
 ARG BUILD_PROFILE=""
+ARG CONF_PATH=""
 
 COPY --from=build /build/server/service /server/service
-COPY --from=build /build/config/server${BUILD_PROFILE}.yaml /etc/tigrisdata/tigris
+COPY --from=build /build/${CONF_PATH}config/server${BUILD_PROFILE}.yaml /etc/tigrisdata/tigris
```

Comment:
Should we add some validation for these paths?

Reply:
Not sure what we can do here. If file doesn't exists copy will fail.